app/jobs/regular/emit_web_hook_event.rb

```diff
@@ -46,6 +50,22 @@ def web_hook_request
       end
 
       conn.post(headers: headers, body: body)
+
+      _, id, event_name = name.split('.')
+      web_hook_event = WebHookEvent.find(id)
```

Comment:
Why do we create an instance of `web_hook_event` at the start and retrieve it later? I think we should only create the `web_hook_event` after `POST`ing the results